data/json/items/fuel.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "chem_ethanol",
+    "type": "AMMO",
+    "name": "ethanol",
+    "name_plural": "ethanol",
+    "description": "Extremely pure alcohol suitable for use as a component in chemical reactions.  You could also water it down and drink it, or use it as fuel in alcohol-burning stoves.",
+    "weight": 1,
+    "volume": 1,
+    "price": 200,
+    "price_post": 50,
```

Comment:
Ethanol sounds like one of those things that wouldn't lose much of its value post apocalypse. There would be no tax on it, but it would have higher demand. Price 200 for both pre and post would be fine.

Also, it's `price_postapoc`, not just `post`.